modules/components/src/DropDown/DropDown.js

```diff
@@ -76,32 +76,32 @@ class DropDown extends React.Component {
                   right: align === 'right' ? 0 : 'auto',
                   left: align === 'right' ? 'auto' : 0,
                 }}
+                {...(singleSelect && { onClick: this.handleToggleMenu })}
```

Comment:
close the menu upon selecting an option, if no multiple selection is available (i.e. checkboxes)